package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "react-native-music-control",
-  "version": "0.10.6",
+  "version": "0.10.8",
```

```suggestion
  "version": "0.10.6",
```

no need to update that